nodes/sample_host.json

```diff
@@ -53,7 +53,8 @@
         "username": "<db username, max 10 characters>",
         "password": "<enter a random password>",
         "database": "<appname>_<stage>"
-      }
+      },
+      "gzip_enabled": true
```

Comment:
Wouldn't it be better to have the sample default to `false`? Since people do have to perform some task on their rails apps to make this work